packages/core/mesh/network-manager/src/swarm/connection.ts

```diff
@@ -255,7 +298,7 @@ export class Connection {
       }
 
       // If signal fails treat connection as failed
-      log.warn('Signal failed', { err });
+      log.warn('Signal failed, closing', { err });
```

Comment:
Nits:
- our log message standard is generally terse and lower case `log.warn('signal failed', { err })`
- this.close already logs "closing..."